Update architecture-connections.md (#103327)
Azure/azure-content: articles/defender-for-iot/organizations/architecture-connections.md

# OT sensor cloud connection methods

> This article describes the architectures and methods supported for connecting your Microsoft Defender for IoT OT sensors to the cloud. An integral part of the Microsoft Defender for IoT service is the managed cloud service in Azure that acts as the central security monitoring portal for aggregating security information collected from network monitoring sensors.OT network sensors connect to Azure to provide alert and device data and sensor health messages, access threat intelligence packages, and more. Connected Azure services include IoT Hub, Blob Storage, Event Hubs,Aria and the Microsoft Download Center.In order to ensure the security of IoT/OT at a global scale, the service supports millions of concurrent telemetry sources securely and reliably. 

Reviewer: ```suggestion
This article describes the architectures and methods supported for connecting your Microsoft Defender for IoT OT sensors to the Azure portal in the cloud. 

OT network sensors connect to Azure to provide data about detected devices, alerts, and sensor health messages, to access threat intelligence packages, and more. Connected Azure services include IoT Hub, Blob Storage, Event Hubs, Aria, the Microsoft Download Center, and more.  
```
